webpack.config.js

```diff
@@ -188,11 +188,11 @@ if (isProd) {
       }
     },
     {
-      'react-addons-css-transition-group': {
-        root: ['React','addons','CSSTransitionGroup'],
-        commonjs2: 'react-addons-css-transition-group',
-        commonjs: 'react-addons-css-transition-group',
-        amd: 'react-addons-css-transition-group',
+      'react-transition-group': {
+        root: ['React', 'addons', 'CSSTransitionGroup'],
+        commonjs2: 'react-transition-group',
+        commonjs: 'react-transition-group',
+        amd: 'react-transition-group',
       }
```

Comment:
Can you change this to 

```js
  {
      'react-transition-group/CSSTransitionGroup': {
        root: ['React', 'addons', 'CSSTransitionGroup'],
        commonjs2: 'react-transition-group/CSSTransitionGroup',
        commonjs: 'react-transition-group/CSSTransitionGroup',
        amd: 'react-transition-group/CSSTransitionGroup',
      }
    }
```
Otherwise the whole package is included in the build file which significantly increases the file size.

Everything else looks good and I'll merge once the changes are done.